src/s3contents.ts

```diff
@@ -905,14 +882,15 @@ export class Drive implements Contents.IDrive {
     }
 
     // save file with new content by overwritting existing file
-    await this._s3Client.send(
+    const promise = await this._s3Client.send(
```

Comment:
If you already `await` here, you do not need to create the `promise` constant or `await` it below.
```suggestion
    await this._s3Client.send(
```